.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = false
```

Comment:
@AureliaDolo

Reply:
why ?

Reply:
Why don't you want final newlines ?

Reply:
I changed it, it was the default value of the config file generated by VSCode!

Reply:
It doesn't appear as changed